todo-list/src/App.js

```diff
@@ -0,0 +1,13 @@
+import TodoList from "./components/TodoList";
+import AddToDo from "./components/AddToDo";
```

Comment:
Not perfect circled button

<img width="111" alt="Screenshot 2023-01-16 at 22 12 28" src="https://user-images.githubusercontent.com/34886027/212758814-1d91882f-adc4-4494-9b64-492b85076a04.png">